### ISSUE TITLE
GoogleHome not refresh air conditioner entities

### ISSUE THREAD

**Issue by developerGM** [Andre0512/hon#235]
I am using v.0.14.0beta7 and I have tested with stable version v.0.13 but if I try to use GoogleHome
"Hey Google, please poweroff air conditioner in the kitchen"
The air conditioner power off but the enty in HomeAssistant not refresh, result always in ON

Other info:
HA Core: 2024.7.3
Supervisor: 2024.06.2
IPv4 static
IPv6 disabled

**Comment by Pakstyle** [comment]
I have the same issue, i got home automations on them Homeassistant keeps telling me air conditioner is on but its off

**Comment by jacamac** [comment]
It seems that sometimes the state is not refreshed until you make another change

**Comment by mercedsl** [comment]
from version 0.14.0 all sensors will not update autonomously, this prevents normal use of any automation, until the previous release it worked perfectly

**Comment by PetrMa** [comment]
Yes, I have the same problem with my wash machine on 0.14.0. all sensors will not update autonomously

**Comment by jfmcarreira** [comment]
For me I also noticed that the version 0.14 does not update the state of Air conditioner when I change in the remote. 

I reverted back to 0.13 to use pooling

**Comment by galvani** [comment]
check https://github.com/Andre0512/hon/issues/269
